build/mediaelement-and-player.js

```diff
@@ -2407,6 +2406,81 @@ window.MediaElement = mejs.MediaElement;
 
 }(mejs.i18n.locale.strings));
 /*!
+ * This is a i18n.locale language object.
```

Comment:
where does this whole block of I18n stuff come from?  do we have a previous commit that neglected to do the build before merging?